src/main/java/io/github/moulberry/notenoughupdates/listener/ItemTooltipRngListener.java

```diff
@@ -59,6 +61,8 @@ public class ItemTooltipRngListener {
 
 	private final Pattern SLAYER_INVENTORY_TITLE_PATTERN = Pattern.compile("(.+) RNG Meter");
 
+	private final Pattern TIER_AMOUNT = Pattern.compile("§5§o§7Tier (.+) amount: §a(\\d+).*");
```

Comment:
We should start thinking about what name design we want field patterns to have. Both types have been used a similar number of times throughout the project. (THIS_FORMAT or thatFormat)

Reply:
I prefer using uppercase with underscores for constant like variables such as patterns, since I do believe that is the official naming convention for constant fields